verify-loader

```diff
@@ -155,9 +162,11 @@ def verify(input_gen, report_interval=REPORT_INTERVAL):
                         invocid = invocid.strip()
                         ctx = contexts[invocid]
                         prev = ctx.msg(seq, line_len)
+                        # check for out of order lines - this implies LOG LOSS
                         if prev is not None:
                             # Bad record encountered, flag it
                             print("%s: %d %d  <-" % (invocid, seq, prev))
+                            loss_count += (seq-prev)
```

Comment:
So if `seq` is less than `prev` this will be a negative value.  I don't think we want to account for loss that way.

Instead we might want to consider two conditions: `seq > prev + 1` and `seq <= prev`.

The add the distance between `seq` and `prev` makes sense on the first condition.  But for the second condition we'll want to keep track of contiguous ranges, expanding the range as the sequence grows, creating a new sequence when a gap is encountered, and looking for duplicates by seeing if the new `seq` is in any known ranges.